WordPress/src/main/java/org/wordpress/android/ui/comments/EditCommentActivity.java

```diff
@@ -309,22 +309,15 @@ private void cancelEditCommentConfirmation() {
             return;
         }
 
-        AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(
-                new ContextThemeWrapper(this, R.style.Calypso_Dialog_Alert));
+        MaterialAlertDialogBuilder dialogBuilder = new MaterialAlertDialogBuilder(this);
         dialogBuilder.setTitle(getResources().getText(R.string.cancel_edit));
         dialogBuilder.setMessage(getResources().getText(R.string.sure_to_cancel_edit_comment));
         dialogBuilder.setPositiveButton(getResources().getText(R.string.yes),
-                new DialogInterface.OnClickListener() {
-                    public void onClick(DialogInterface dialog, int whichButton) {
-                        finish();
-                    }
-                });
+                (dialog12, whichButton) -> finish());
         dialogBuilder.setNegativeButton(
                 getResources().getText(R.string.no),
-                new DialogInterface.OnClickListener() {
-                    public void onClick(DialogInterface dialog, int whichButton) {
-                        // just close the dialog
-                    }
+                (dialog1, whichButton) -> {
+                    // just close the dialog
```

Comment:
Same as in a previous comment, maybe I would use `dialog` (instead of `dialog1`)